bigtable/integration_test.go

```diff
@@ -2250,16 +2250,47 @@ func TestIntegration_AdminBackup(t *testing.T) {
 
 	adminClient, err := testEnv.NewAdminClient()
 	if err != nil {
-		t.Fatalf("NewAdminClient: %v", err)
+		t.Errorf("NewAdminClient: %v", err)
 	}
 	defer adminClient.Close()
 
-	table := testEnv.Config().Table
-	cluster := testEnv.Config().Cluster
-
+	tblConf := TableConf{
+		TableID: "tableconf-backup",
```

Comment:
Should this still be taken from testEnv?

Reply:
Do you mean the table id? Just curious, what's the benefit there?

Reply:
Yep-- it seems best to have these configs live in a single place for all tests so that they can be easily modified/swapped.

Reply:
got it thanks!